src/Command/StartChainCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Command;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class StartChainCommand extends Command
+{
+    private LoggerInterface $logger;
+
+    private const CHAIN_INFO = '%s registered as a member of %s command chain';
+
+    public const EXECUTE_INFO = 'Executing %s from chain members';
+
+    protected static $defaultName = 'app:start-chain';
+
+    protected static $defaultDescription = 'Runs chain of commands';
+
+    private string $commandChainId;
+
+    public function __construct(string $commandChainId, LoggerInterface $logger, string $name = null)
+    {
+        $this->commandChainId = $commandChainId;
+        $this->logger = $logger;
+
+        parent::__construct($name);
+    }
+
+    public function configure()
+    {
+        parent::configure();
+        $this->setDescription(
+            '%s is a master command of a command chain that has registered member commands'
```

Comment:
as the command already has $defaultDescription specified, we don't need to override it here.